.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+**Description**
+linked issues, what is implemented or fixed, how to test it, …
```

Comment:
suggest to add

linked companion PRs if this PR is not compatible with the backend dev branch

Reply:
changed